sources/osgUtil/TriangleLineSegmentIntersector.js

```diff
@@ -5,7 +5,7 @@ var mat4 = require( 'osg/glMatrix' ).mat4;
 var TriangleIndexFunctor = require( 'osg/TriangleIndexFunctor' );
 var ComputeMatrixFromNodePath = require( 'osg/computeMatrixFromNodePath' );
 
-var TriangleIntersection = function ( normal, i1, i2, i3, r1, r2, r3 ) {
+var TriangleLineSegmentIntersection = function ( normal, i1, i2, i3, r1, r2, r3 ) {
```

Comment:
Don't agree in changin that name. I would stick to TriangleIntersector, I don't see any good reason to chnage the OSG standard. Also TriangleIntersector is a utility class, users shouldn't be using this directly. In fact in OSG is inside a cpp and not extensible at all.